dataland-frontend/tests/e2e/specs/index.ts

```diff
@@ -24,6 +24,8 @@ require("./infrastructure");
 if (runPrepopulation) {
   if (!singlePopulate || cypressTestGroup === 1 || cypressTestGroup === 101) {
     require("./prepopulation");
+  } else if (cypressTestGroup === 3) {
```

Comment:
Is this only for logging or does test group 3 now run before prepopulation is finished?